mount/mount.c

```diff
@@ -98,8 +145,25 @@ int main() {
 		}
 	}
 
+	/* For NFS, we must do the dns resolution *here* while we are inside net ns */
+	if (nfs_mount_hostname) {
+		printf("Decoding and resolving dns hostname");
+		static struct sockaddr_in server_addr;
+		static char ip_addr[16] = "127.0.0.1";
```

Comment:
Should this be initialized at all?

Reply:
I guess it is a little defensive against something going wrong and having it crash.
I guess in our case crashing would probably be better than weird behavior where it tries to connect to localhost.